public/app/features/live/data/amendTimeSeries.ts

```diff
@@ -1,4 +1,4 @@
-import { closestIdx } from "./StreamingDataFrame";
+import { closestIdx } from '@grafana/data/src/dataframe/StreamingDataFrame';
```

Comment:
nit: import from `@grafana/data` instead.

Reply:
same, it's not exported either